code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -128,7 +128,7 @@
 	desc = "A card used to provide ID and determine access across the station."
 	icon_state = "id"
 	item_state = "card-id"
-	var/access = list()
+	var/list/access = list()

```

Comment:
Are you certain this won't generate issues anywhere, from legacy code treating access as a string which is later converted into a list?